src/main/resources/i18n/messages.properties

```diff
@@ -500,7 +500,13 @@ TrTab2.26=Keep AC-3 tracks
 TrTab2.27=Use LPCM for audio
 TrTab2.28=Keep DTS tracks
 TrTab2.29=AC-3 re-encoding audio bitrate: (in Kbps) (e.g. 576, 640)
+TrTab2.30=Keep LPCM tracks
```

Comment:
It would be better if new strings used words instead of numbers like `TrTab2.KeepLPCMTracks`

Reply:
I do not mind such a change. Do you do so?

Reply:
I can do it